Cargo.toml

```diff
@@ -1,38 +1,43 @@
 [package]
 name = "psoc6-pac"
-version = "0.0.0"
-authors = ["Jonas Schievink <jonasschievink@gmail.com>"]
+version = "0.0.1"
+authors = ["Jonas Schievink <jonasschievink@gmail.com>", "Dave Humphries <humphriesdj@gmail.com"]
 edition = "2018"
 description = "Peripheral Access Crate for Cypress PSoC 6 MCUs"
 documentation = "https://docs.rs/psoc6-pac/"
-repository = "https://github.com/jonas-schievink/psoc6-pac.git"
+repository = "https://github.com/diagonalDave/psoc6-pac.git"
```

Comment:
We can probably just point at `https://github.com/psoc-rs/psoc6-pac` here and use the `psoc-rs` org link for other spots too.